commercial-paper/roles/network-clean.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: Apache-2.0
+
+function _exit(){
+    printf "Exiting:%s\n" "$1"
+    exit -1
+}
+
+# Exit on first error, print all commands.
+set -ev
+set -o pipefail
+
```

Comment:
I would remove the identity artifacts as well 
rm -rf organization/digibank/identity/user/balaji/wallet/*
rm -rf organization/magnetocorp/identity/user/isabella/wallet/*